evaluation_plans/osps/build_release/evaluations.go

```diff
@@ -174,3 +174,33 @@ func OSPS_BR_06() (evaluation *layer4.ControlEvaluation) {
 
 	return
 }
+
+func OSPS_BR_07() (evaluation *layer4.ControlEvaluation) {
+	evaluation = &layer4.ControlEvaluation{
+		ControlID: "OSPS-BR-07",
+	}
+
+	evaluation.AddAssessment(
+		"OSPS-BR-07.01",
+		"A bunch of control requirement text that should not be duplicated here but is needed to make the control valid.",
```

Comment:
harsh 😆 

There has been discussion around using this space to summarize the test instead of writing out the control requirement, but that's not currently the end-user expectation.